backends/tfhe-cuda-backend/cuda/include/integer.h

```diff
@@ -300,14 +300,16 @@ template <typename Torus> struct int_radix_lut {
     // PBS
     if (params.pbs_type == MULTI_BIT) {
       // Only 64 bits is supported
-      static_assert(
-          sizeof(Torus) == 8,
-          "Error (GPU multi bit PBS): only 64 bits Torus is supported");
-      scratch_cuda_multi_bit_pbs_64(
-          stream, &pbs_buffer, params.small_lwe_dimension,
-          params.glwe_dimension, params.polynomial_size, params.pbs_level,
-          params.grouping_factor, num_radix_blocks,
-          cuda_get_max_shared_memory(stream->gpu_index), allocate_gpu_memory);
+      if (sizeof(Torus) == sizeof(uint32_t)) {
```

Comment:
I think a better solution would be 

```C++
if (sizeof(Torus) == sizeof(uint64_t)) {
// Run the operation
} else {
PANIC("...")
}
```

This way we also handle the possibility of someone running it for `uint8_t` and `uint16_t` (and who knows what else).